src/component/parser/json/converter/DiagramConverter.ts

```diff
@@ -140,20 +148,28 @@ export default class DiagramConverter extends AbstractConverter<BpmnModel> {
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private deserializeLabel(bpmnLabel: any, id: string): Label {
-    const labelStyle = bpmnLabel?.labelStyle;
-    if (labelStyle) {
-      const font = this.findFont(labelStyle);
+    if (bpmnLabel) {
+      const labelStyle = bpmnLabel.labelStyle;
+      const font = this.findFont(labelStyle, id);
+      const bounds = this.deserializeBounds(bpmnLabel);
 
-      if (font) {
-        return new Label(font);
-      } else {
-        // TODO error management
-        console.warn('Unable to assign font %s to shape/edge %s', labelStyle, id);
+      if (font || bounds) {
+        return new Label(font, bounds);
       }
     }
   }
 
-  private findFont(labelStyle: string): Font {
-    return this.convertedFonts.get(labelStyle);
+  private findFont(labelStyle: string, id: string): Font {
+    let font;
+    if (labelStyle) {
+      font = this.convertedFonts.get(labelStyle);
+    }
+
+    if (!font) {
+      // TODO error management
+      console.warn('Unable to assign font from style %s to shape/edge %s', labelStyle, id);
+    }
```

Comment:
:warning: I suggest to move this block in the `if (labelStyle)` block.
When `labelStyle` is `undefined` (ie when no bpmn label has been specified), `font` is always `undefined`, so we generate a warning whereas this is something valid.
In addition, it is very common to not define label for all or most bpmn elements, so we currently generate a lot of inaccurate console warnings
Notice that
- these extra warnings have not been introduced by this PR, I just detect this during the PR review
- the propose change may require to update tests

Reply:
No updating test.
It was a mistake during the refactoring ;)

Reply:
Done